tasks/notify.py

```diff
@@ -179,7 +180,24 @@ async def run_async_within_lock(
                 "Not sending notifications yet because we are waiting for CI to finish",
                 extra=dict(repoid=commit.repoid, commit=commit.commitid),
             )
-            if commit.repository.using_integration or commit.repository.hookid:
+            ghapp_default_installations = list(
```

Comment:
I see this affects the notify task for coverage uploads. Would it be tough to add the GH App integration for the bundle analysis notifier?  https://github.com/codecov/worker/blob/c248c6f78f6ac8d4ef954ea49f394241da46b060/services/bundle_analysis.py#L268

Reply:
(If unfamiliar, BA uses a different notifier cause the normal notifier relied too much on coverage data being present, hence Scott opted to do a separate notifier wrapper class)

Reply:
When making these changes I looked for references of `.using_integration`. There was no such ref in the bundle analysis notifier.

I guess it was decided that that task doesn't need to check this for the timeout on the retry. Personally I think that this implementation here is a bit "too much", but was respecting the logic that's in place.